src/components/BareMetalHosts/BaremetalHostStatus.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getHostStatus } from '../../utils/status/host/hostStatus';
+import { GenericStatus, GenericError, GenericProgress, GenericSuccess, ValidationError } from './StatusComponents';
+import {
+  HOST_STATUS_READY,
+  HOST_STATUS_DISCOVERED,
+  HOST_STATUS_OK,
+  HOST_STATUS_PROVISIONED,
+  HOST_STATUS_REGISTERING,
+  HOST_STATUS_INSPECTING,
+  HOST_STATUS_PREPARING_TO_PROVISION,
+  HOST_STATUS_PROVISIONING,
+  HOST_STATUS_DEPROVISIONING,
+  HOST_STATUS_MAKING_HOST_AVAILABLE,
+  HOST_STATUS_VALIDATION_ERROR,
+  HOST_STATUS_REGISTRATION_ERROR,
+} from '../../utils/status/host/constants';
+
+// Map status strings to components
+const componentMap = {
```

Comment:
why is this not the same as `HOST_STATUS_ALL_SUCCESS, HOST_STATUS_ALL_PROGRESS` ? Can we use these? 

If not, can you implement one of these options?

1. move this logic to `host/hostStatus.js` ? and return something like 

```
{
 ...start,
 genericStatus: HOST_GENERIC_STATUS_SUCCESS,
}
```
from there?

\+ create constants for progress/success/validationError, etc. ?  And then do the mapping of these constants here? Ik we would still neeed to do the mapping here but it would be less verbose and more extensible.

2. option is to create another array something like `HOST_GENERIC_STATUS_ALL_SUCCESS`